### PR TITLE
Added linux support for tool

### PR DESCRIPTION
**Pull request by AndreSteenbergen**
Added the directory separator, instead of hard coded back slash

### CODE REVIEW

File: Source/Schema.NET/core/WebPageElement.cs [hunk 1]
```diff
@@ -15,5 +15,19 @@ public partial class WebPageElement : CreativeWork
         /// </summary>
         [DataMember(Name = "@type", Order = 1)]
         public override string Type => "WebPageElement";
+
+        /// <summary>
+        /// A CSS selector, e.g. of a &lt;a class="localLink" href="http://schema.org/SpeakableSpecification"&gt;SpeakableSpecification&lt;/a&gt; or &lt;a class="localLink" href="http://schema.org/WebPageElement"&gt;WebPageElement&lt;/a&gt;. In the latter case, multiple matches within a page can constitute a single conceptual "Web page element".
+        /// </summary>
+        [DataMember(Name = "cssSelector", Order = 206)]
+        [JsonConverter(typeof(ValuesConverter))]
+        public OneOrMany<CssSelectorType>? CssSelector { get; set; }
```
Comment (RehanSaeed) [inline Source/Schema.NET/core/WebPageElement.cs]: I guess that you did not run the tool to generate these two properties. Maintaining this by hand is not really scalable due to the number of types. However, the tool project has some overrides that add extra properties that don't exist in schema.org but are defined in Google structured data.

File: Source/Schema.NET.Tool/Services/SchemaService.cs [hunk 2]
```diff
@@ -102,7 +103,7 @@ private static void CombineMultipleParentsIntoCombinedClass(List<Class> classes)
                             Description = classDescription,
                             Id = new Uri($"http://CombinedClass/{className}"),
                             IsCombined = true,
-                            Layer = @class.IsCombined ? @class.Layer : $"{@class.Layer}\\combined",
+                            Layer = @class.IsCombined ? @class.Layer : $"{@class.Layer}{Path.DirectorySeparatorChar}combined",
```
Comment (RehanSaeed) [inline Source/Schema.NET.Tool/Services/SchemaService.cs]: This is all good stuff and I can merge this if it was a separate PR.

File: Source/Schema.NET/core/WebPageElement.cs (hunk 1 above)
Comment (AndreSteenbergen) [reply Source/Schema.NET/core/WebPageElement.cs]: I actually ran the tool as is, and these came up automatically. I actually added those 2 files by hand, as these were not generated (they are basically strings)

**Comment by AndreSteenbergen** [comment]
This is actually just a phase 1 commit; a larger project is coming to allow mixed types, without adding a whole bunch of interfaces (you can check out Values<T1>). I think it won't be that hard. just that jsonconvertor ..... :( .

File: Source/Schema.NET/core/WebPageElement.cs (hunk 1 above)
Comment (RehanSaeed) [reply Source/Schema.NET/core/WebPageElement.cs]: See https://github.com/RehanSaeed/Schema.NET/tree/master/Source/Schema.NET.Tool/Overrides for examples of how to add stuff like this.

**Comment by RehanSaeed** [comment]
Small PR's make things easier to merge. The directory separator can be a quick and easy merge.

**Comment by RehanSaeed** [comment]
Thanks for your work by the way!

**Comment by AndreSteenbergen** [comment]
Will make a PR that is smaller, and also check out that link thx

**Comment by AndreSteenbergen** [comment]
I'll close this one, and reopen another, easiest for me.

**Pull request closed without merging**